extensions/wikia/PortableInfobox/templates/PortableInfoboxItemGroup.mustache

```diff
@@ -1 +1 @@
-<section class="pi-item pi-group pi-border-color">{{{content}}}</section>
+<section class="pi-item pi-group pi-border-color{{#cssClasses}} {{{cssClasses}}}{{/cssClasses}}">{{{content}}}</section>
```

Comment:
I think it could be - I just wanted to avoid that not needed space in case cssClasses is empty.